frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigForm.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import InputLabel from '@material-ui/core/InputLabel';
+import { makeStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import uuid from "uuid";
+import Typography from '@material-ui/core/Typography';
+import Tooltip from '@material-ui/core/Tooltip';
+
+{/**TO DO: Fix issue with setting state */}
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    backgroundColor: theme.palette.background.paper,
+    margin: theme.spacing(1),
+    minWidth: 250,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+
+export default function ConfigForm({ addConfig }) {
+  const classes = useStyles();
+
+  const [config, setConfig] = useState("");
+
+  const dimensionsMetrics = ["noodle", "spice", "egg", "soup", "instant noodle"];
+  const rDimensionsMetrics = ["noodle", "spice", "egg", "soup", "instant noodle"];
```

Comment:
make variable name more clear (ie "relatedDimensionsMetrics") instead of just 'rDimensionsMetrics'